vaadin-combo-box-flow-parent/vaadin-combo-box-flow/src/test/java/com/vaadin/flow/component/combobox/ComboBoxTest.java

```diff
@@ -55,18 +54,6 @@ public class ComboBoxTest {
     @Rule
     public ExpectedException thrown = ExpectedException.none();
 
-    private static class TestComboBox extends ComboBox<String> {
-
-        private List<String> items;
-
-        @Override
-        public void setDataProvider(ListDataProvider<String> dataProvider) {
-            super.setDataProvider(dataProvider);
-            items = new ArrayList<>(dataProvider.getItems());
-        }
-
-    }
```

Comment:
This was used in a test to verify that a `ListDataProvider` was set when calling a specific `setItems` method, and then check the items from it. This doesn't work anymore, because `ComboBoxBase` now delegates all calls to the DataController, so this method is never called.
Changed the test below to just get the current DataProvider and then cast it to `ListDataProvider`